scripts/gen_csv.py

```diff
@@ -0,0 +1,197 @@
+        self.Venous = None
+
+
+dic = {} 
+
+# read the first training data
+tree = ElementTree.parse('./text1.xml')
+root = tree.getroot()
+docs  = root.findall('docs')[0]
+for doc in docs:
+    text_data = doc.findall('text')[0].text  # extract texxt data
+    pat = Patient() # create pateint
+    pat.text = text_data
+    pat.id = int(doc.attrib['id'])
+    dic[pat.id] = pat
+
+# read the second training data
+tree = ElementTree.parse('./text2.xml')
+root = tree.getroot()
+docs  = root.findall('docs')[0]
+for doc in docs:
+    text_data = doc.findall('text')[0].text  # extract texxt data
+    pat = Patient() # create pateint
+    pat.text = text_data
+    pat.id = int(doc.attrib['id'])
+    dic[pat.id] = pat
+
+# read the annotations 1
+tree = ElementTree.parse('./label1.xml')
+root = tree.getroot()
+diseases  = root.findall('diseases')[0]
+for disease in diseases:
+    dis_name = disease.attrib['name'].split(' ')[0]
+    print(dis_name)
+    for doc in disease:
+        id_ = int(doc.attrib['id'])
+        setattr(dic[id_] , dis_name, doc.attrib['judgment'])
+
+# read the annotations 2
+tree = ElementTree.parse('./label2.xml')
+root = tree.getroot()
+diseases  = root.findall('diseases')[0]
+for disease in diseases:
+    dis_name = disease.attrib['name'].split(' ')[0]
+    for doc in disease:
+        id_ = int(doc.attrib['id'])
+        setattr(dic[id_] , dis_name, doc.attrib['judgment'])
+
+# read the annotations 3
+tree = ElementTree.parse('./label3.xml')
+root = tree.getroot()
+diseases  = root.findall('diseases')[0]
+for disease in diseases:
+    dis_name = disease.attrib['name'].split(' ')[0]
+    for doc in disease:
+        id_ = int(doc.attrib['id'])
+        setattr(dic[id_] , dis_name, doc.attrib['judgment'])
+
+
+d_train = {
+    'id' : [],
+    'text' : [],
+    'Asthma': [],
+    'CAD': [],
+    'CHF': [],
+    'Depression': [],
+    'Diabetes': [],
+    'Gallstones': [],
+    'GERD': [],
+    'Gout': [],
+    'Hypercholesterolemia': [],
+    'Hypertension': [],
+    'Hypertriglyceridemia': [],
+    'OA': [],
+    'Obesity': [],
+    'OSA': [],
+    'PVD': [],
+    'Venous' : [],
+}
+
+for pat in dic.values():
+    d_train['id'].append(pat.id)
+    d_train['text'].append(pat.text)
+    d_train['Asthma'].append(pat.Asthma)
+    d_train['CAD'].append(pat.CAD)
+    d_train['CHF'].append(pat.CHF)
+    d_train['Depression'].append(pat.Depression)
+    d_train['Diabetes'].append(pat.Diabetes)
+    d_train['Gallstones'].append(pat.Gallstones)
+    d_train['GERD'].append(pat.GERD)
+    d_train['Gout'].append(pat.Gout)
+    d_train['Hypercholesterolemia'].append(pat.Hypercholesterolemia)
+    d_train['Hypertension'].append(pat.Hypertension)
+    d_train['Hypertriglyceridemia'].append(pat.Hypertriglyceridemia)
+    d_train['OA'].append(pat.OA)
+    d_train['Obesity'].append(pat.Obesity)
+    d_train['OSA'].append(pat.OSA)
+    d_train['PVD'].append(pat.PVD)
+    d_train['Venous'].append(pat.Venous)
+
+data_train = pd.DataFrame(d_train)
```

Comment:
If you add the above as a method on the `Patient` class, we can do this whole thing like this:

```
data_train = pd.DataFrame([pat.get_attrs() for pat in dic.values()])
```

`get_attrs()` could look something like:

```
def get_attrs(self):
    return {
        'id': self.id,
        'text': self. text,
        ...
    }
```

That way it's cleaner and we can also re-use it for the test set below.